cluster/cluster.go

```diff
@@ -718,6 +718,11 @@ func InitClusterObject(ctx context.Context, rkeConfig *v3.RancherKubernetesEngin
 	if len(c.CertificateDir) == 0 {
 		c.CertificateDir = GetCertificateDirPath(c.ConfigPath, c.ConfigDir)
 	}
+	// Setting cluster Defaults
+	err = c.setClusterDefaults(ctx, flags)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
We need `isEncryptionEnabled` to only be called after `c.setClusterDefaults(ctx, flags)` to ensure that `c.EncryptionConfig.EncryptionProviderFile` is set by default for a k8s 1.18 cluster here:

https://github.com/rancher/rke/blob/ea15c3d861aa3a2fc1d130b6f73e84021b760dc9/cluster/cluster.go#L731-L734